src/CommunityToolkit.Maui.Core/Handlers/DrawingView/DrawingViewHandler.shared.cs

```diff
@@ -180,79 +180,4 @@ void OnPlatformViewLinesCollectionChanged(object? sender, NotifyCollectionChange
 		VirtualView.Lines.CollectionChanged += OnVirtualViewLinesCollectionChanged;
 	}
 }
-
```

Comment:
Can we remove #if block or replace it with #if !net?

Reply:
We could remove the `#if` block by moving the code to `DrawingViewHandler.macios.cs`, `DrawingViewHandler.android.cs` and `DrawingViewHandler.windows.cs`, but then we'd have duplicated ~150 lines of code in each new file. 

And we can't use `#if !NET` because that is also used by `net-ios`, `net-maccatalyst`, `net-android` and `net-windows`
<img width="937" alt="image" src="https://user-images.githubusercontent.com/13558917/188553245-da9a61ae-c6f7-41cb-89aa-1be3e7243b20.png">

Reply:
I think this is the best way to do it to reduce code duplication. 

Plus, if we ever add support for a future TFM, e.g. `net6.0-tizen`, this current `#if IOS || ...` implementation will ensure a build error happens here which then ensures we remember to verify `DrawingViewHander` specifically for the new TFM.

Reply:
Let me know what you think! If you're cool with it we can Approve/Merge this PR 🙌

Reply:
Agree, the less code we duplicate, the more benefit we get from cross-platform development.